app/src/main/java/com/onewelcome/showcaseapp/feature/userregistration/browserregistration/BrowserRegistrationViewModel.kt

```diff
@@ -143,4 +163,8 @@ class BrowserRegistrationViewModel @Inject constructor(
     data class UseDefaultIdentityProvider(val isChecked: Boolean) : UiEvent
     data class HandleRegistrationCallback(val uri: Uri) : UiEvent
   }
+
+  sealed class NavigationEvent {
+    object ToPinScreen : NavigationEvent()
```

Comment:
AS suggests it can be converted to `data object ToPinScreen : NavigationEvent()`